lib/external/math.dart

```diff
@@ -40,4 +50,128 @@ ASTNode funcRand(Runtime runtime, ASTNode self, List<ASTNode> args) {
   }
 
   throw const UnexpectedTypeException('The rand method only takes [String, double, int, and bool] argument types');
+}
+
+ASTNode funcSin(Runtime runtime, ASTNode self, List<ASTNode> args){
+  runtimeExpectArgs(args, [ASTType.AST_ANY]);
+  final radian = args[0];
+  
+  if(radian is IntNode || radian is DoubleNode){
+    final DoubleNode result = DoubleNode()..doubleVal = sin(radian is DoubleNode ? radian.doubleVal : radian.intVal);
+    return result;
+  }
+  
+  throw const UnexpectedTypeException('The sin method only accept [double, int] argument types');
```

Comment:
`[double, int]` is misleading as it could be interpreted as a `List`

Reply:
I've changed `[double, int]` to `either double or int`